gateway/api/openapi/types.go

```diff
@@ -30,8 +30,10 @@ type (
 )
 
 const (
-	StatusActive   StatusType = "active"
-	StatusInactive StatusType = "inactive"
+	StatusActive    StatusType = "active"
+	StatusInactive  StatusType = "inactive"
+	StatusReviewing StatusType = "reviewing"
```

Comment:
Do we have these values duplicated?
I will comment on the duplicates.

Reply:
I don't fully support having them duplicated, but one is for OpenAPI documentation spec, and the other is internally used. We could use one, but we are using both. We can change that on another refactor, so we don't go changing all around, as it's not the focus of this PR

Reply:
@sandromello instructed me on removing it, so it's done. Thanks for bringing it up